javascript/ql/src/semmle/javascript/dataflow/Sources.qll

```diff
@@ -193,10 +193,13 @@ module SourceNode {
   /**
    * A data flow node that is considered a source node by default.
    *
-   * Currently, the following nodes are source nodes:
+   * This includes all nodes that evaluate to a new object and all nodes whose
+   * value is computed using non-local data flow (that is, flow between functions,
+   * between modules, or through the heap):
+   *
    *   - import specifiers
    *   - function parameters
-   *   - `this` nodes
+   *   - `this` expressions
```

Comment:
`ThisExpr` is not a source node. Maybe say `function receivers` instead?

Reply:
Makes sense.

Reply:
Changed.